sc/managers/configloaders/common/admin.go

```diff
@@ -3,22 +3,83 @@ package common
 import (
 	"encoding/json"
 
+	"github.com/mitchellh/mapstructure"
+	"github.com/spacecloud-io/space-cloud/config"
+	"github.com/spacecloud-io/space-cloud/model"
 	"github.com/spf13/viper"
 )
 
-func prepareAdminApp() json.RawMessage {
+func prepareAdminApp(fileConfig *model.SCConfig) json.RawMessage {
 	user := viper.GetString("admin-user")
 	pass := viper.GetString("admin-pass")
 	secret := viper.GetString("admin-secret")
 	isDev := viper.GetBool("dev")
 
 	config := map[string]interface{}{
-		"user":   user,
-		"pass":   pass,
-		"secret": secret,
-		"isDev":  isDev,
+		"user":     user,
+		"pass":     pass,
+		"secret":   secret,
+		"isDev":    isDev,
+		"projects": prepareAdminProjects(fileConfig),
 	}
 
 	data, _ := json.Marshal(config)
 	return data
 }
+
+func prepareAdminProjects(fileConfig *model.SCConfig) map[string]*config.AdminProject {
+	adminProjects := make(map[string]*config.AdminProject)
+	module, ok := fileConfig.Config["adminman"]
+	if !ok {
+		return adminProjects
+	}
+	resourceObjects, ok := module["config"]
+	if !ok {
+		return adminProjects
+	}
+
+	for _, resourceObject := range resourceObjects {
+		adminConfig := new(config.AdminProjectConfig)
+		if err := mapstructure.Decode(resourceObject.Spec, adminConfig); err != nil {
+			return map[string]*config.AdminProject{}
+		}
+
+		projectID := resourceObject.Meta.Parents["project"]
```

Comment:
This won't have a parent. The name of the object will be the project id

Reply:
we are calling it project in admin module

Reply:
There is no projects